packages/base/card-api.gts

```diff
@@ -844,7 +846,7 @@ class LinksTo<CardT extends CardConstructor> implements Field<CardT> {
     return value;
   }
 
-  async handleNotLoadedError<T extends Card>(
+  async handleNotLoadedError<T extends CardBase>(
```

Comment:
linksTo cards are never primitive, perhaps this is a Card type.

Reply:
I think I can't change this due to inheritance